web/src/games/tictactoe/locales/pt.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Jogo da velha",
+  "description": "Um jogo clássico",
+  "descriptionTag": "Jogue o Jogo da Velha (ou jogo do galo) online gratuitamente.\\\nVocê pode jogar nos modo de jogador úncio contra o computador,\\\nmodo de multi-jogador contra um amigo online, ou compartilhar\\\nseu dispositivo e jogar localmente contra um amigo.\n",
```

Comment:
jogo do galo? didnt know that one :P. Is it portuguese from portugal?

Reply:
hahaha yeah! I was reading Wikipedia and found out!